npm/src/controller/oauth.ts

```diff
@@ -48,9 +48,30 @@ import { SSOHandler } from './sso-handler';
 import { ValidateOption, extractSAMLResponseAttributes } from '../saml/lib';
 import { oidcClientConfig } from './oauth/oidc-client';
 import { App } from '../ee/identity-federation/app';
+import * as encrypter from '../db/encrypter';
+import { Encrypted } from '../typings';
 
 const deflateRawAsync = promisify(deflateRaw);
 
+function encrypt(val: any) {
```

Comment:
Could keep this functions under `./oauth` folder.

Reply:
Not used anywhere else so it's fine to be local to this file